apps/admin/backend/src/app.ts

```diff
@@ -49,12 +54,18 @@ function getCurrentElectionDefinition(
   return mostRecentlyCreatedElection?.electionDefinition;
 }
 
-function constructDippedSmartCardAuthMachineState(
+function constructAuthMachineState(
   workspace: Workspace
 ): DippedSmartCardAuthMachineState {
   const currentElectionDefinition = getCurrentElectionDefinition(workspace);
   return {
     electionHash: currentElectionDefinition?.electionHash,
+    // TODO: Pull jurisdiction from VxAdmin cert authority cert
+    jurisdiction: isFeatureFlagEnabled(
+      BooleanEnvironmentVariableName.ENABLE_JAVA_CARDS
+    )
+      ? /* istanbul ignore next */ DEV_JURISDICTION
+      : undefined,
```

Comment:
Falling back to `undefined` when `ENABLE_JAVA_CARDS` is `false` just makes sure that development with memory cards is unaffected by this check that doesn't apply to them